modules/migration-updating-deprecated-internal-images.adoc

```diff
@@ -46,6 +46,17 @@ $ podman login -u $(oc whoami) -p $(oc whoami -t) --tls-verify=false <registry_u
 $ podman pull <registry_url>:<port>/openshift/<image>
 ----
 
+. Scan the {product-title} 3 image for deprecated namespaces:
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command), steps that introduce a command need to have it explicitly stated somewhere:
```suggestion
. Scan the {product-title} 3 image for deprecated namespaces by running the following command:
```